adapter/slack/adapter.go

```diff
@@ -1,477 +1,39 @@
-/*
- * Copyright 2021 The Gort Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 package slack
```

Comment:
This file should have automatically picked up a copyright notification. 
Can you force run the [pre-commit](https://pre-commit.com/) hooks with a `pre-commit run -a` ?

This will also fix line endings and clean up any extra whitespaces.